### PR TITLE
Basic info parcel

### PR DESCRIPTION
**Pull request by joeldenning**

### CODE REVIEW

File: src/dashboard-widgets/basic-info/basic-info.parcel.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    )
+  }
+
+  function renderIdentifiers() {
+    return patient.identifiers.map(identifier => `${identifier.identifierType.name}: ${identifier.identifier}`).join(' - ')
+  }
+}
+
+type BasicInfoParcelProps = {
+  patientId: string,
```
Comment (fatmali) [inline src/dashboard-widgets/basic-info/basic-info.parcel.tsx]: @joeldenning we are giving it the patient uuid and not the id, perhaps we could rename this variable to patientUuid

File: src/dashboard-widgets/dashboard-widgets.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {css} from '@emotion/core'
+import Parcel from 'single-spa-react/parcel'
+
+const patientDashboardParcels = [
+  () => import('./basic-info/basic-info.parcel').then(m => m.default),
+]
+
```
Comment (fatmali) [inline src/dashboard-widgets/dashboard-widgets.tsx]: @joeldenning why are we using promises with the import statement?

File: src/dashboard-widgets/basic-info/basic-info.parcel.tsx (hunk 1 above)
Comment (joeldenning) [reply src/dashboard-widgets/basic-info/basic-info.parcel.tsx]: 👍 updated

File: src/dashboard-widgets/dashboard-widgets.tsx (hunk 2 above)
Comment (joeldenning) [reply src/dashboard-widgets/dashboard-widgets.tsx]: https://github.com/tc39/proposal-dynamic-import and https://webpack.js.org/guides/code-splitting/#dynamic-imports

Comment (joeldenning) [reply src/dashboard-widgets/dashboard-widgets.tsx]: https://github.com/CanopyTax/single-spa-react#parcel-props

**Pull request merged**